Jenkinsfile

```diff
@@ -8,8 +8,8 @@ pipeline {
         // It is recommended by SQUARE team do not add the label and let the
         // system decide.
         docker {
-            image 'lsstsqre/centos:w_latest'
-            args '-u root'
+          image 'lsstts/develop-env:develop'
```

Comment:
Why do you want to use `lsstts/develop-env:develop` instead of `lsstsqre/centos:w_latest`? The package manager in `ts_ofc` is `eups` instead of `conda` now.

Reply:
I saw the following in the history:
```
Use develop-env image for the CI job, due to the need of java to build the documentation.
```

It looks like the lsst image does not support this anymore. So sad...